opm/output/eclipse/Summary.cpp

```diff
@@ -1353,11 +1353,22 @@ quantity roew(const fn_args& args) {
     return { oil_prod / args.initial_inplace.get( region_name, Opm::Inplace::Phase::OIL, args.num ) , measure::identity };
 }
 
-template< bool injection = true>
-inline quantity temperature( const fn_args& args ) {
-    const quantity zero = { 0, measure::temperature };
-    if (args.schedule_wells.empty())
+template <bool injection = true>
+quantity temperature(const fn_args& args)
+{
+    const quantity zero = {
+        // Note: to_si(0.0) to properly handle different temperature
+        // offsets.  This value will convert back to 0.0 of the appropriate
+        // unit when we later call .from_si().  The plain value 0.0 here
```

Comment:
consider a slight rewording. 'If a plain value 0.0 is used here, it will be treated....'

Reply:
> consider a slight rewording. 'If a plain value 0.0 is used here, it will be treated....'

Good idea.  I've pushed an update to this effect.